mastracode/web/src/web/ui/domains/auth/components/sign-in-page.css

```
@@ -0,0 +1,103 @@
+.factory-signin-theme {
+  --surface1: #080808;
+  --surface2: #0f0f0f;
+  --surface3: #171717;
+  --surface4: #1c1c1c;
+  --surface5: #262626;
+  --surface6: #303030;
+  --neutral1: #424242;
+  --neutral2: #686868;
+  --neutral3: #939393;
+  --neutral4: #cccccc;
```

Stylelint (17.14.0)

[error] Expected "`#cccccc`" to be "`#ccc`" (color-hex-length)

(color-hex-length)